addons/hr_timesheet/models/hr_timesheet.py

```diff
@@ -312,3 +312,20 @@ def _get_timesheet_time_day(self):
     def _employee_timesheet_cost(self):
         self.ensure_one()
         return self.employee_id.timesheet_cost or 0.0
+
+    @api.model
+    def ensure_uom_hours(self):
```

Comment:
That function can be private since we don't want to call it in client (JS code) 🙂